src/Microsoft.TestPlatform.CommunicationUtilities/Serialization/TestObjectConverter.cs

```diff
@@ -55,10 +55,10 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
         // key value pairs.
         foreach (var property in properties)
         {
-            var testProperty = property["Key"].ToObject<TestProperty>(serializer);
+            var testProperty = property["Key"]!.ToObject<TestProperty>(serializer)!;
 
             // Let the null values be passed in as null data
-            var token = property["Value"];
+            var token = property["Value"]!;
```

Comment:
Same question here (i.e. above you are checking for null token and here not )

Reply:
nope. Fixed.